testplan/common/exporters/__init__.py

```diff
@@ -43,6 +43,8 @@ class BaseExporter(Configurable):
 
     def __init__(self, **options):
         self._cfg = self.CONFIG(**options)
+        self.export_path = None
+        self.name = self.__class__.__name__
```

Comment:
Would it be more tp2 style if we define a "name" config option in ExporterConfig?